content/posts/euc/hub-templates.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Hub Service Template"
+date: 2020-10-24T00:00:00-00:00
+draft: false
+categories: [HUBSERVICES, EUC]
+author: Nagul Subramanian
+authorlink: /https://www.linkedin.com/in/nagulsubramanian 
+tags:
+- Access
+- Hub Services
+- Templates 
+- Workspace ONE
+lightgallery: true
+---
+
+## What is Hub Services? 
+To simply state it helps enterprise introduce a true digital experience to their employees on any device or web. This is just getting started with lot of potential to unlock features to make their employee experience a world class. 
+
+There are few docs that captures all the features in Hub Services in detail as referenced below, so with that I want to focus specifically on the Hub Services template in this post. 
+- [Enabling and Delivering Hub Services with Workspace ONE Intelligent Hub](https://kb.vmware.com/s/article/2960170)
+- [VMware Workspace ONE Intelligent Hub Series Part 1: Evolve from Legacy App Catalog as Web Clip to Hub Catalog](https://thomascheng.net/2020/07/17/evolve-from-legacy-app-catalog-as-web-clip-to-hub-catalog-within-vmware-workspace-one-intelligent-hub/)
+## Why we need the Hub Services Templates?
+
+- To truly provide an user-centric experience we need a way to personalize/tailor content for each personas. Let's take the knowledge worker and the sales personas as an example, each of their day-to-day functions differ and rely on unique resources to accomplish their task. Such that it is important to provide a curated experience and that can achieved via the Hub Services template. As the name represents, it allows you to create templates with different set of features and/or configurations and can then be deployed to UEM smart groups or Access user groups or both 
+
+{{< admonition type=note title="Note" open=true >}}
+- UEM smart groups assignment - applicable for iOS, Android, macOS and Windows platform 
+- Access user group assignment - applicable for Web experience (require Access integration and Access to set as Auth) 
+{{< /admonition >}}
+
+## Setup
+
+If you haven't configured the Hub Services, please follow the instructions here: [Activating Hub Services](https://kb.vmware.com/s/article/2960170).
+
+In this post, as previously referenced, I'm taking the knowledge worker and the sales personas as an example to create a template for each. The common use case I see in the field is, admin creates template(s) for testing feature(s) with a small group of users prior to the Global rollout. 
```

Comment:
should the second sentence be - "another common use case that I see in the field..." ?

Reply:
Something else to consider is to summarize what we are trying to enable for knowledge worker vs sales so everyone can follow along. 
Knowledge worker - Branding 1, People, Custom Tab with a certain URL
Sales - Branding 2, People, Customer tab with different URL 
if you can show a screenshot or preview of the experience for both personas, that would help everyone visualize and follow along.